kdoctor/src/macosMain/kotlin/org/jetbrains/kotlin/doctor/entity/System.kt

```diff
@@ -83,6 +83,26 @@ actual fun System.execute(command: String, vararg args: String): ProcessResult =
         retCode.value
     }
 
+    if (verbose) {
```

Comment:
Is this just for debugging purposes or there are other applications?

Reply:
Debugging solely